pkg/statistics/handle/util/util.go

```diff
@@ -249,16 +249,16 @@ func GetFullTableName(is infoschema.InfoSchema, tblInfo *model.TableInfo) string
 
 // JSONTable is used for dumping statistics.
 type JSONTable struct {
-	Columns           map[string]*JSONColumn `json:"columns"`
-	Indices           map[string]*JSONColumn `json:"indices"`
-	Partitions        map[string]*JSONTable  `json:"partitions"`
-	DatabaseName      string                 `json:"database_name"`
-	TableName         string                 `json:"table_name"`
-	ExtStats          []*JSONExtendedStats   `json:"ext_stats"`
-	Count             int64                  `json:"count"`
-	ModifyCount       int64                  `json:"modify_count"`
-	Version           uint64                 `json:"version"`
-	IsHistoricalStats bool                   `json:"is_historical_stats"`
+	Columns           map[string]*JSONColumn `json:"columns,omitempty"`
```

Comment:
Can I ask why we need to use omitempty here? (You don't want zero value in your JSON file?)  I guess this might affect the JSON output of this struct. I am not sure if this is OK. Hope nobody marshals it by themself or depends on the structure of the JSON object.

Reply:
Not save space obviously.